src/index.html

```diff
@@ -19,9 +19,382 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
Stylesheets with the .scss extension won't work directly in browsers. You should compile them to .css files and then link to the compiled .css file in your HTML.